### PR TITLE
[OSD-11456] Address gosec lint warnings

### PR DESCRIPTION
**Pull request by a7vicky**
### What this PR does / why we need it?
The PR addresses the gosec lint warnings

### Which Jira/Github issue(s) this PR fixes?

_Fixes #_[OSD-11456](https://issues.redhat.com/browse/OSD-11456)

### Pre-checks (if applicable):
- [ ] Tested latest changes against a cluster
- [ ] Included documentation changes with PR

### CODE REVIEW

File: util/access_token.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	pullSecretKey     = ".dockerconfigjson"
+	pullSecretKey     = ".dockerconfigjson" //#nosec G101 -- This is a false positive
```
Comment (bmeng) [inline util/access_token.go:16]: I am wondering if we should fix the "confusing" name of the const.

And it is interesting there the next line is not being caught?

**Comment by bmeng** [comment]
The fix looks good to me.
One only comment, maybe we should enable the gosec linter as a default one in this repo, as we have the golangci.yml already.

**Comment by a7vicky** [comment]
> The fix looks good to me. One only comment, maybe we should enable the gosec linter as a default one in this repo, as we have the golangci.yml already.

Thanks @bmeng for the review. I have enabled gosec linter in golangci.yml

**Pull request merged**